service/src/test/java/bio/terra/externalcreds/dataAccess/WriteTransactionProbe.java

```diff
@@ -0,0 +1,80 @@
+package bio.terra.externalcreds.dataAccess;
+
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.springframework.dao.CannotSerializeTransactionException;
+import org.springframework.dao.RecoverableDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class WriteTransactionProbe {
```

Comment:
Since this isn't a test itself, can we move it into, like, a `util` subfolder or something? I assumed it was one because of its location.

Reply:
I disagree that you should make that assumption. This is an area for tests _and_ test related code.